developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst

```diff
@@ -8,6 +8,13 @@ General
 Front-end changes
 -----------------
 
+Breaking changes
+^^^^^^^^^^^^^^^^
+
```

Comment:
should this get its own heading?

Reply:
Would you put it directly into the one above?

Reply:
replace *Breaking changes* with *Font sizes* and the structure looks good imo

there might be other breaking changes in 30, that's why I would just not put the paragraph as *Breaking changes*

Reply:
Done :white_check_mark: